.cloudbuild/cloudbuild.yaml

```diff
@@ -55,6 +55,10 @@ steps:
           "gcr.io/cloud-devrel-public-resources/graalvm_b:${_JAVA_SHARED_CONFIG_VERSION}",
       ]
     waitFor: [ "graalvm-b-build" ]
+
+# The docker images will be published to GCP projects cloud-devrel-kokoro-resources and cloud-devrel-public-resources
+# following convention in https://github.com/googleapis/testing-infra-docker. While the former will be used for Kokoro
```

Comment:
I don't see https://github.com/googleapis/testing-infra-docker explaining the two locations.

Reply:
@suztomo Unfortunately, there isn't explicit documentation for it in the repo's README. Received context for it from patterns observed in other language Cloud Build triggers (https://github.com/googleapis/testing-infra-docker/blob/f3a27546b3a9fedef75a78f22c8fde23d5c013d3/python/cloudbuild.yaml#L61-L69, https://github.com/googleapis/testing-infra-docker/pull/215) and internal documentation on the "cloud client libraries gcp projects" page

Reply:
Can you start with the public one only, as we don't store any credentials there?

If you find something needs non-public one, we learn something new.